package.json

```diff
@@ -9,9 +9,11 @@
   "packageManager": "pnpm@10.6.4",
   "scripts": {
     "build": "pnpm -r build",
-    "package:latest": "node deploy/publish --tag latest",
-    "package:next": "node deploy/publish --tag dev",
-    "preinstall": "npx only-allow pnpm"
+    "build:package": "pnpm --filter='./packages/**' -r build",
+    "package:latest": "pnpm --filter='./packages/**' -r publish --tag latest",
+    "package:next": "pnpm --filter='./packages/**' -r publish --tag next",
+    "preinstall": "npx only-allow pnpm",
+    "release": "bumpp"
```

Comment:
When is this command executed?

Reply:
I'll add the documentation later

Reply:
https://github.com/wrtnlabs/agentica/pull/183
I wrote it in this PR!

Reply:
thx you for your PR's!